.travis.yml

```diff
@@ -1,28 +1,48 @@
-# Use Travis's cointainer based infrastructure
-sudo: false
+sudo: required
+dist: trusty
 
 addons:
   apt:
     sources:
-      - chef-current-precise
+      - chef-stable-trusty
```

Comment:
We can test on current again now that we don't delete the promoted build out of current when we take a build to stable.